easybuild/easyconfigs/p/Python/Python-3.6.4-foss-2018a.eb

```diff
@@ -216,6 +216,10 @@ exts_list = [
         'source_urls': ['https://pypi.python.org/packages/source/m/mpmath'],
         'checksums': ['04d14803b6875fe6d69e6dccea87d5ae5599802e4b1df7997bddd2024001050c'],
     }),
+    ('tabulate', '0.8.2', {
```

Comment:
@wpoely86 Please open a separate PR for this, and make this consistent for all (non-bare) Python 3.6.4 easyconfigs + all more recent (3.x) versions.

Reply:
sure

Reply:
@wpoely86 This PR can't go in until we have a PR merged that adds `tabulate` to all recent Python 3.x easyconfigs...

Reply:
https://github.com/easybuilders/easybuild-easyconfigs/pull/6809